### PR TITLE
fix a bug in MPI bcast linsys_max

### PR DESCRIPTION
**Pull request by shenmidelin**
a bug is fixed in src/io.f90

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,4 @@
 # List required packages in this file, one per line.
+numpy
+scipy
+matplotlib
```
Comment (mpmdean) [inline requirements.txt]: I think `sympy` might need to be here too?

Comment (shenmidelin) [reply requirements.txt]: Yes, we need sympy. I will add it.

**Comment by mpmdean** [comment]
HI @mrakitin and @shenmidelin 

Can I get write access to the repo?

**Comment by mrakitin** [comment]
@mpmdean and @shenmidelin are invited to https://github.com/orgs/NSLS-II/teams/edrixs-team.

**Pull request merged**